src/Team.test.jsx

```diff
@@ -13,4 +13,37 @@ describe('<Team />', () => {
     const wrapper = shallow(<Team />);
     expect(wrapper.get(0)).toMatchSnapshot();
   });
-});
\ No newline at end of file
+
+  it('has a Button that opens the SkillModal', () => {
+    const wrapper = mount(<Team />);
+    const skillModal = wrapper.find('Modal')
+      .findWhere(n => n.prop('contentLabel') == 'AddTeamMemberModal');
```

Comment:
I like this approach. I have tried using other `enzyme` functions besides `findWhere` that would be more succinct and clear, but couldn't get them working (no idea why). So I resorted to doing a string search with the `debug` function, but this is a much better approach 👍